projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -792,6 +799,10 @@ private boolean answerType(
 
    private boolean cat(String[] words)
          throws IOException, FileNotFoundException {
+      if (words.length == 1) {
+         _logger.errorf(words[0] + " requires additional arguments\n");
+         return false;
+      }
```

Comment:
the error here is not if the length is 1. The error here is if the length != 2. Make sure the length check rejects invalid commands with *too many* arguments also.